projects/asthma-copd/nbs/30_gls_phenoplier/05-summarize.ipynb

```diff
@@ -0,0 +1,2293 @@
+{
```

Comment:
<p> </p><p>A comment at the top to broadly describe its function and how it is expected to be used (e.g. imported, run from command line, both).  &lt;&lt;&lt;&lt; from review checklist</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/phenoplier/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Do you mean that the current description should be extended? The explanation on how to run the notebook (both from the browser and the command line) is in the <code>README.md</code> file in the root directory of the repository. I'm happy to add more details if needed!</p>